api/db/changelog/db.changelog-61-replace-idv-with-beta-access.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="jroberti" id="db.changelog-61-replace-idv-with-beta-access">
+        <dropColumn tableName="user">
+            <column name="beta_access_completion_time"/>
+            <column name="requested_id_verification"/>
+            <column name="id_verification_request_time"/>
+        </dropColumn>
+        <addColumn columnName="beta_access_request_time" tableName="user" type="datetime"/>
```

Comment:
We're dropping a fair amount of data here. I'm not sure whether it will cause any issues but it's probably safer to rename the id_verification_request_time column instead.